plasma_framework/docs/audits/README.md

```diff
@@ -0,0 +1,9 @@
+## Audits 
+
+The following list contains information on all the audits that have been conducted for the Plasma contracts component of the OMG network. 
+
+| Release | Organization | Period | Report |
+|--|--|--|--|
+| [v0.0.1](https://github.com/omisego/plasma-contracts/releases/tag/v0.0.1) | Quantstamp | 24.09.2018 - 10.11.2018 | [Report](https://github.com/omisego/plasma-contracts/blob/v0.0.1/docs/quantstamp-audit-3cc6097.md)|
```

Comment:
Do you mind if we use ISO date format instead? `yyyy-mm-dd` 

https://www.iso.org/iso-8601-date-and-time-format.html

Reply:
ok